imagedephi/redact/redact.py

```diff
@@ -37,6 +38,19 @@ def iter_image_files(directory: Path) -> Generator[Path, None, None]:
             yield child
 
 
+def create_redact_dir(base_output_dir: Path) -> Path:
+    """Given a directory return a timestamped directory created in the base directory."""
+    time_stamp = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+    redact_dir = base_output_dir / f"Redacted_{time_stamp}"
+    try:
+        redact_dir.mkdir(parents=True)
+        click.echo(f"Created redaction folder: {redact_dir}")
+    except PermissionError:
+        click.echo("Cannnot create an output directory, permission error.")
```

Comment:
What's the intended control flow after this point? We're catching an exception and allowing `return redact_dir` to run normally afterwards, but we know the directory doesn't actually exist.